src/components/menu/examples/menu-open-direction.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+    MenuItem,
+    ListSeparator,
+    OpenDirection,
+    Option,
+} from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ *
+ */
```

Comment:
Something like "Here you can try different positions of the menu"

Reply:
(I copied your first comment about this into your comment above, and hid the original. Just so that it's all in this thread that we can resolve later 🙂)